Source/DafnyDriver/Commands/DocCommand.cs

```diff
@@ -0,0 +1,50 @@
+using System.Collections.Generic;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.Linq;
+
+namespace Microsoft.Dafny;
+
+class DocCommand : ICommandSpec {
+
+  public static readonly Option<string> DocProgramNameOption = new("--doc-program-name",
```

Comment:
I would remove the prefix `doc`, from these option names, since these options are specific to the `doc` command.

Reply:
Done